CHANGELOG.md

```diff
@@ -1,40 +1,76 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## 0.12.0 (July 29, 2022)
+
+### Enhancements
+ * Support for `@TemplateGlobal` annotation. See [#605](https://github.com/redhat-developer/quarkus-ls/issues/605).
+ * Support for `textDocument/InlayHint` in Qute templates. See [#595](https://github.com/redhat-developer/quarkus-ls/issues/595).
+ * Delay revalidation of Java files in Qute language server. See [#666](https://github.com/redhat-developer/quarkus-ls/pull/666).
+ * Delay revalidation after edit and improve cancel checking. See [#659](https://github.com/redhat-developer/quarkus-ls/pull/659).
+ * Improve memory and peformance of Qute language server. See [#654](https://github.com/redhat-developer/quarkus-ls/issues/654).

```

Comment:
Maybe it would be worthwhile to put these under a "performance" section?

Reply:
Yup, I'll do that. I ended up combining the first two entries into 1.